src/Swift.Runtime/src/Library/UnsafePointer.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Swift.Runtime
+{
+    // <summary>
+    // Represents Swift UnsafePointer in C#.
+    // </summary>
+    public unsafe readonly struct UnsafePointer<T>
+    {
+        private readonly void* _rawValue;
+        public UnsafePointer(void* _rawValue)
```

Comment:
The argument name should not have underscore. Also, should the argument name match the property name that it corresponds to `UnsafePointer(void* poin_rawValue)`?